ui/app/src/modules/Preview/QuickCreate.tsx

```diff
@@ -0,0 +1,160 @@
+import MenuItem from '@material-ui/core/MenuItem';
+import Divider from '@material-ui/core/Divider';
+import { createStyles, makeStyles, Theme } from '@material-ui/core/styles';
+import { palette } from "../../styles/theme";
+import { getQuickCreateContentList } from '../../services/content';
+import { useActiveSiteId, useSpreadState, useSelection } from '../../utils/hooks';
+import EmbeddedLegacyEditors from './EmbeddedLegacyEditors';
+
+const translations = defineMessages({
+  quickCreateBtnLabel: {
+    id: 'quickCreateBtnLabel.label',
+    defaultMessage: 'Open quick create menu'
+  }
+});
+
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    addBtn: {
+      fill: palette.green.main,
+      width: 30,
+      height: 30,
+      "&:hover": {
+        fill: palette.green.shade
+      }
+    },
+
+    menu: {
+      transform: 'translate(20px, 15px)',
+      "& ul": {
+        paddingTop: 0,
+        minWidth: '140px'
+      }
+    },
+
+    menuItem: {
+      fontSize: 14,
+    },
+
+    menuTitle: {
+      fontSize: 14,
+      "&:hover": {
+        cursor: 'text',
+        backgroundColor: 'transparent'
+      }
+    },
+
+    menuSectionTitle: {
+      fontSize: 12,
+      backgroundColor: palette.gray.light0,
+      color: palette.gray.medium3,
+      paddingTop: 2,
+      paddingBottom: 2,
+      "&:hover": {
+        backgroundColor: palette.gray.light0,
+        cursor: 'text'
+      }
+
+    },
+  })
+);
+
+export default function QuickCreate() {
+  const classes = useStyles({});
+  const { formatMessage } = useIntl();
+  const siteId = useActiveSiteId();
+  const AUTHORING_BASE = useSelection<string>(state => state.env.AUTHORING_BASE);
+  const defaultFormSrc = `${AUTHORING_BASE}/legacy/form`;
+  const [anchorEl, setAnchorEl] = useState(null);
+  const [quickCreateContentList, setQuickCreateContentList] = useState(null);
+  const [dialogConfig, setDialogConfig] = useSpreadState({
+    open: false,
+    src: defaultFormSrc,
+    type: 'form',
+    inProgress: false,
+  });
+
+  const handleClick = e => setAnchorEl(e.currentTarget);
+
+  const handleMenuClose = () => setAnchorEl(null);
+
+  useEffect(() => {
+    if(siteId) {
+      getQuickCreateContentList(siteId).subscribe(data => 
+        setQuickCreateContentList(data.items)
+      )
+    }
+  }, [siteId])
+
+  const handleFormDisplay = srcData => {
+    const { contentTypeId, path } = srcData;
+    const today = new Date();
+    const formatPath = path.replace('{year}/{month}', `${today.getFullYear()}/${today.getFullYear()}`)
+    const src = 
+      `${defaultFormSrc}?newEdit=article&contentTypeId=${contentTypeId}&path=${formatPath}&type=form`;
+
+    setDialogConfig({
+      open: true,
+      src
+    })
+  }
+
+
+  return (
+    <>
+      <IconButton 
+        onClick={handleClick} 
+        aria-label={formatMessage(translations.quickCreateBtnLabel)}
+      >
+        <AddCircleIcon fontSize="small" className={classes.addBtn}/>
```

Comment:
Should the class `addBtn` be rather on the IconButton? Otherwise rename accordingly to signal it is indeed for the icon.